### DISCUSSION TITLE
Intermediate HTML and CSS: Move framworks lesson to Advanced section

### DISCUSSION THREAD

**Discussion by i4pg** [TheOdinProject/curriculum#25599] [General]
### Describe your suggestion

Intermediate HTML CSS section has frameworks lesson, It's not clear that it isn't recommended, Which may indicate some confusion on deciding whether to incorporate CSS frameworks.

I've worked all my way up through rails path, Now on JS section incorporating Bulma.io, it was good and fast for developing though that's lead to some lost of CSS touches

No regret at all. But, for better learning experience I believe this lesson should be at the last of advanced CSS HTML

### Path

Other / NA

### Lesson Url

 Frameworks and Preprocessors for both Rails and JS paths

### Checks

- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### (Optional) Discord Name

i4pg#4897

### (Optional) Additional Comments

I had a conversation with @LeoKirasic on odin-offtopic room, Thanks to him for pointing that out to me

**Comment by impronen** [comment]
I'm coming in as someone on the JS path, so my experience has been a bit different but still I agree with this. 

The question about should I learn this or that framework during the JS frontend section comes up frequently enough on the server and most always is met with advice to "not yet" by the more experienced folks over there. 

I was advised to wait for Tailwind until at least to React and I've seen similar things echoed relatively frequently. 

But then again, as the question is posted often (especially by people on the Js path) there is a case to be made for some mention of frameworks at the end of the Intermediate HTML / CSS lessons. Without any guidance people might be _more_ included to dive into framework rabbit holes than with the current lesson content where it is. 

So I'm supportive (for what it's worth) but I do think some mention is needed.

**Comment by codyloyd** [comment]
I think our reason for putting them _there_ is that by this point most people are already aware of them, and we get a lot of questions about them in our help channels.

I wouldn't be against adding a more strongly worded disclaimer to the lesson, telling people not to use them for the CSS lessons... but I don't think we want to move them.

I'm going to convert this issue to a discussion so we can hash it out before making a decision.

**Comment by codyloyd** [comment]
@TheOdinProject/intermediate-html-css 

what do y'all think?

**Comment by thatblindgeye** [comment]
@wise-king-sullyman and I talked about this and came to the conclusion that we agree with what @codyloyd said above. If having stronger verbiage would help make it clearer that using frameworks/preprocessors isn't recommended we can certainly take in a PR that does that.
